spring-integration-ip/src/test/java/org/springframework/integration/ip/tcp/TcpConfigOutboundGatewayTests-context.xml

```diff
@@ -37,6 +38,9 @@
 							   serializer="crLfSerializer"
 							   deserializer="crLfSerializer"/>
 
+	<ip:tcp-inbound-gateway id="gatewayCrLf2"
+		connection-factory="crLfServer2"
+		request-channel="toSA" />
```

Comment:
Perviously, the server used the same port as `crLfServerNio` (7400) and `crLfServer2` had no gateway so it was never started. The result was that this test actually talked to the other inbound gateway.